### PR TITLE
Secure Camera

### PR DESCRIPTION
**Pull request by dngray**
Adding Graphene OS's secure Camera.

The EXIF removal stuff seems useful.

### CODE REVIEW

File: _data/operating-systems/android-applications/4_secure_camera.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+title: Secure Camera
+type: Recommendation
+logo: /assets/img/android/secure_camera.svg
+description: |
+  **Secure Camera** is an camera app focused on privacy and security which can capture images, videos, and QR codes. CameraX vendor extensions (Portrait, HDR, Night Sight, Face Retouch and Auto) are also supported on available devices.
```
Comment (mfwmyfacewhen) [inline _data/operating-systems/android-applications/4_secure_camera.yml:5]: Add comma after Retouch for consistency

**Review by mfwmyfacewhen — CHANGES_REQUESTED** [review]

File: _data/operating-systems/android-applications/4_secure_camera.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+title: Secure Camera
+type: Recommendation
+logo: /assets/img/android/secure_camera.svg
+description: |
+  **Secure Camera** is an camera app focused on privacy and security which can capture images, videos, and QR codes. CameraX vendor extensions (Portrait, HDR, Night Sight, Face Retouch, and Auto) are also supported on available devices.
+
+  Main privacy features include
+  - Auto stripping of [Exif](https://en.wikipedia.org/wiki/Exif) metadata (enabled by default)
+  - Use of the new [Media](https://developer.android.com/training/data-storage/shared/media) API, therefore [storage permissions](https://developer.android.com/training/data-storage) are not required
+  - Microphone permission not required unless you want to record sound
```
Comment (ph00lt0) [inline _data/operating-systems/android-applications/4_secure_camera.yml:10]: Should be unless you record video. At least from my experience you can't record video without sound.

**Review by ph00lt0 — CHANGES_REQUESTED** [review]

File: _data/operating-systems/android-applications/4_secure_camera.yml (hunk 2 above)
Comment (ph00lt0) [reply _data/operating-systems/android-applications/4_secure_camera.yml:10]: @flexagoon found it now, got some help elsewhere. That is a very hidden feature, maybe would be worth to add some further explanation: "In order to record video without sound you have to disable 'Include Audio' this in the video settings, available under the tool-tip on top of the screen."

Comment (flexagoon) [reply _data/operating-systems/android-applications/4_secure_camera.yml:10]: @ph00lt0 It's not a hidden feature. When you first install the app it doesn't request the microphone permission. And if you try to record a video, it shows a warning saying that audio is disabled and you need to enable 'Include Audio' in the settings.

**Pull request merged**